goalie/mesh_seq.py

```diff
@@ -324,7 +325,9 @@ def get_checkpoints(
                 checkpoints.append(
                     AttrDict(
                         {
-                            field: project(sols[field], fs[i + 1])
+                            field: firedrake.Function(fs[i + 1]).interpolate(
```

Comment:
Two comments:
1. I think we should still support `project`, with an option to switch. I guess `interpolate` should be the default, if we can get it to work better in parallel.
2. Do we expect the `interpolate` method to work for mixed spaces? (I haven't tested it.) The `project` method doesn't, which is why we had to write an interpolation module for Goalie.